api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -667,16 +683,12 @@ private boolean userHasEverLoggedIn(
   public ResponseEntity<Void> updateProfile(Profile updatedProfile) {
     DbUser user = userProvider.get();
 
-    // Save current profile for audit trail. Continue to use the userProvider (instead
-    // of info on previousProfile) to ensure addition of audit system doesn't change behavior.
```

Comment:
can't use the userProvider in the common code because it may refer to the admin user instead of the update target